ksqldb-engine/src/test/java/io/confluent/ksql/function/udf/math/ExpTest.java

```diff
@@ -39,22 +40,22 @@ public void shouldHandleNull() {
 
   @Test
   public void shouldHandleNegative() {
-    assertThat(udf.exp(-1), is(0.36787944117144233));
-    assertThat(udf.exp(-1L), is(0.36787944117144233));
-    assertThat(udf.exp(-1.0), is(0.36787944117144233));
+    assertThat(udf.exp(-1), is(closeTo(0.36787944117144233, 0.00000000000000005)));
+    assertThat(udf.exp(-1L), is(closeTo(0.36787944117144233, 0.00000000000000005)));
+    assertThat(udf.exp(-1.0), is(closeTo(0.36787944117144233, 0.00000000000000005)));
   }
 
   @Test
   public void shouldHandleZero() {
-    assertThat(udf.exp(0), is(1.0));
-    assertThat(udf.exp(0L), is(1.0));
-    assertThat(udf.exp(0.0), is(1.0));
+    assertThat(udf.exp(0), is(closeTo(1.0, 0.1)));
```

Comment:
Do these udfs really have epsilon around 0.1? Can't we use 5E-17 similar to other cases?

Reply:
Thanks for the feedback, @Gerrrr !
I've improved it with your suggestion! Fully agreed that it is easier to read now! 
I've changed it to `1E-15`. Otherwise someone later may wonder why `5` and not `3` or `8`.